compiler/src/dotty/tools/dotc/parsing/JavaParsers.scala

```diff
@@ -521,7 +521,7 @@ object JavaParsers {
           val vparams = formalParams()
           if (!isVoid) rtpt = optArrayBrackets(rtpt)
           optThrows()
-          val bodyOk = !inInterface || (mods.is(Flags.DefaultMethod))
+          val bodyOk = !inInterface || mods.is(Flags.DefaultMethod) || mods.is(Flags.JavaStatic)
```

Comment:
Thank you for looking at this, perhaps it can be simplified to
```suggestion
          val bodyOk = !inInterface || mods.isOneOf(Flags.DefaultMethod | Flags.JavaStatic)
```

Reply:
Done